sig-scheduling/charter.md

```diff
@@ -0,0 +1,67 @@
+# SIG Scheduling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+We build Kubernetes schedulers and scheduling features for Pods. We design and
+implement features that allows users to customize placement of Pods on the nodes
+of a cluster. These features include those that improve reliability of workloads,
+more efficient use of cluster resources, and/or enforces placement policies. 
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Scheduling related features (e.g. Node Affinity)
+- Kube-scheduler performance and scalability (with [sig-scalability](../sig-scalability))
+- Kube-scheduler reliability (problem detection and remediation)
+- Pod scheduling APIs (with [sig-api-machinery](../sig-api-machinery))
+- Node-level resource management (with [sig-node](../sig-node))
+- Cluster-level resource management (with [sig-node](../sig-node) and [sig-storage](../sig-storage))
+- Pod scheduling policies (with [wg-policy](../wg-policy))
```

Comment:
SIGs own code, and may sponsor working groups that span sigs.

Is this referring more specifically to `PodDisruptionBudget`?

Reply:
hit enter too soon, meaning to say I would expect this to say "Pod Scheduling Policies (with name other sig)" or be more specific on the resource type if needed.

Reply:
Pod Scheduling Policy is a new concept we are working on with the policy workgroup. I changed the workgroup to sig auth. It defines authorization rules for specifying scheduling requirements, such as tolerations, anti-affinity, etc.